mesh/accounts/views.py

```diff
@@ -44,8 +57,19 @@ def post(self, request):
         try:
             REQUIRED_FIELDS = ['email', 'password', 'phoneNum', 'isMentor', 'isMentee']
             data = validate_json_and_required_fields(request.body, REQUIRED_FIELDS)
-            # data = json.loads(request.body)
-            salt,hash = encrypt(data['password'])
+
+            # check if there is any account created with this email/phoneNum
+            created_email = Account.objects.filter(email = data["email"]).values()
+            created_phone = Account.objects.filter(phoneNum = data["phoneNum"]).values()
+
+            if created_email or created_phone:
+                return JsonResponse(
+                    data={ "status":"account with same email or phone number exists" },
```

Comment:
For the sake of consistency, should the key of this JSONResponse be `error` instead of `status`?